templates/nginx_internal.conf

```diff
@@ -79,6 +77,7 @@ http {
             proxy_pass "http://lemmy";
 
             # proxy common stuff
+            include proxy_params;
             proxy_http_version 1.1;
             proxy_set_header Upgrade $http_upgrade;
             proxy_set_header Connection "upgrade";
```

Comment:
I believe these lines were only needed for websocket. Now that websocket is gone it should be fine to remove them.

Reply:
Just to clarify, the legacy lines are the `proxy_set_header Upgrade $http_upgrade;` and `proxy_set_header Connection "upgrade";`, not the proxy related ones?

Once confirmed, I'll get this fixed up.

Reply:
That's correct, it looks like its those two "upgrade" lines: https://nginx.org/en/docs/http/websocket.html

Reply:
I removed the `proxy_http_version 1.1;` section as well, as I believe that was a requirement of the websockets. Please correct me if I'm wrong.